lib/ad_localize/runner.rb

```diff
@@ -39,10 +51,11 @@ def export(file)
       else
         export_platforms = options.dig(:only) || Constant::SUPPORTED_PLATFORMS
         add_intermediate_platform_dir = export_platforms.length > 1
+        output_path = option_output_path_or_default()
         export_platforms.each do |platform|
           platform_formatter = "AdLocalize::Platform::#{platform.to_s.camelize}Formatter".constantize.new(
             parser.locales.first,
-            options.dig(:output_path),
+            output_path + '/' + output_path_suffix,
```

Comment:
This line will break the change requested in issue #29 . Why using an output_path_suffix ?

Reply:
With no merge management, it's not possible to export multiple csv into one target. 

If `-t` option is used, it's going to export the multiple csv to `<target_directory>/<file_name>/...`

Reply:
Maybe we avoid this obstacle by introducing a rule that says that if there is only one csv to export then there will not be any intermediate directories. Can you try it ?